inst/templates/head.html

```diff
@@ -23,6 +23,8 @@
 {{/yaml}}
 
 <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://cdn.rawgit.com/afeld/bootstrap-toc/v0.4.1/dist/bootstrap-toc.min.css">
+<script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v0.4.1/dist/bootstrap-toc.min.js"></script>
```

Comment:
Would it be possible to add a subresource integrity hash or to locally host these files? I don't feel super comfortable running scripts for an unknown github account, who could change their content to something malicious anytime.